template/.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     needs: formatting
     strategy:
       matrix:
-        os: ['ubuntu-22.04']
+        os: ['ubuntu-22.04', 'windows-latest']
```

Comment:
In the past, we had issues with windows builds taking much longer than linux or osx.
I don't remember if it was a compiling (scipp's C++ code) or a conda issue or something else, but I think we should check build times on various projects before going ahead with this.
It would not be great if everyone's builds just get a lot slower, in addition to the fact that we would be using two builders for the tests instead of 1 (and we only have 20 across the entire organisation).

I can see that building on PRs would be better than a weekly cron job (catch things as early as possible), so if the builds are not too long, we should do it. We should then maybe think again about paying for additional runners?

Reply:
Yes we can make this a weekly cron job if we see things are getting slow.

It shouldn't really take that long with the pure python projects.

Reply:
In my experience with Scitacean (Python only), the Windows jobs are significantly slower than Linux. But MacOs is even slower. So Windows is not such a problem when we don't have to compile C++.

Does the limit of 20 apply across OSs?

Reply:
Maybe we also have to get better with splitting our test suites into short and long tests? In particular in ESSsans there is a fair number of tests on larger files that add up to a significant runtime, and it will likely get worse.

Reply:
> Does the limit of 20 apply across OSs?

Yeah, it's not per repo limit. It depends on the [plan of the organisation.](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits)

Should I just make this a weekly cron job?